models/endoscopic_tool_segmentation/configs/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.4.1",
+    "version": "0.4.2",
     "changelog": {
+        "0.4.2": "support monai 1.2 new FlexibleUNet",
```

Comment:
could you please change `monai_version` into `1.2.0rc2`? Otherwise new flexibleunet will not work

Reply:
Sure, updated, thanks!